components/CodeMirror.vue

```diff
@@ -111,8 +111,12 @@ export default {
 </template>
 
 <style lang="scss">
-  .code-mirror .vue-codemirror .CodeMirror {
-    height: initial;
-    background: none
+  .code-mirror {
```

Comment:
The CodeMirror component is used in a few other places, other than YamlEditor - have you verified  these changes don't cause issues with those uses?

Reply:
Yes ofc, it has been the first thing I noticed.
The other styling are just adjustments related to the layout. I created [this issue](https://github.com/rancher/dashboard/issues/5485) to correct the cross-styling.

![Screenshot from 2022-03-23 13-27-10](https://user-images.githubusercontent.com/5009481/159708256-19c70f48-3199-4979-99a7-7b9861da385b.png)